helm/prometheus-operator/templates/get-tprs-job.yaml

```diff
@@ -32,3 +32,7 @@ spec:
     {{- if .Values.rbacEnable }}
       serviceAccountName: {{ template "fullname" . }}
     {{- end }}
+    {{- if .Values.nodeSelector }}
+      nodeSelector:
+{{ toYaml .Values.nodeSelector | indent 8 }}
```

Comment:
Let's make this specific to hyperkube: `{{ .Values.global.hyperkube.nodeSelector }}`.